packages/ckeditor5-dev-docs/package.json

```diff
@@ -7,9 +7,9 @@
   "dependencies": {
     "@ckeditor/ckeditor5-dev-utils": "^7.0.6",
     "@ckeditor/jsdoc-plugins": "^1.9.3",
-    "jsdoc": "3.4.3",
+    "jsdoc": "3.5.5",
```

Comment:
We cannot upgrade to 3.5. That's why 3.4.3 was hardcoded.

Reply:
Good catch. I forgot about it :/